### DISCUSSION TITLE
Restart Inconsistency Problem

### DISCUSSION THREAD

**Discussion by HappyKian** [ufs-community/ufs-weather-model#1805] [Q&A]
Hi there,

I ran Prototype-P8 atmospheric model (APP=ATM) compiled with the Intel compiler and  CCPP suite FV3_GFS_v16_noahmp on Cheyenne. I tried to restart the model using restart outputs at F06 with the configurations from https://github.com/ufs-community/ufs-weather-model/wiki/Restarting-the-Coupled-Model. But the restart result differed from the one directly integrated from the initial time without pause.

Domain: RRFS_CONUS_13km (pre-generated grid)

I would like to know how to fix this issue. Thank you in advance!

**Comment by jkbk2004** [comment]
@SamuelTrahanNOAA Can I ask some comments about this rrfs restart mismatch issue? Should we consider to coordinate to tackle down the issue with a plan?

**Reply by SamuelTrahanNOAA** [reply]
You can, but we need to involve people who wrote the nesting implementation. It seems to be the commonality between the restart failures. In particular, if someone can try an FV3 restart *without* UFS, using a configuration extremely similar to the UFS, it may give us valuable information.
